superset-frontend/package.json

```diff
@@ -185,10 +182,16 @@
     "redux-undo": "^1.0.0-beta9-9-7",
     "regenerator-runtime": "^0.13.5",
     "rison": "^0.1.1",
-    "shortid": "^2.2.6",
     "urijs": "^1.19.6",
     "use-immer": "^0.4.2",
-    "use-query-params": "^1.1.9"
+    "use-query-params": "^1.1.9",
+    "@data-ui/xy-chart": "^0.0.84",
+    "@superset-ui/chart-controls": "0.17.41",
```

Comment:
The `"@superset-ui/chart-controls": "0.17.41",` is not actually a change, could we make sure to remove diff lines in package.json because it tends to be modified a lot and causes merge issues. Thanks.

Reply:
I had to make those changes to the package.json file because the build was failing.

Reply:
it's fine to add/modify the dependencies, I guess I'm saying try to make the diff contains as little changes as possible. The chart-controls was as line 78 and now it's at 189. But it's the same version. It looks like it was just moved?